plugins/eddn.py

```diff
@@ -1413,14 +1431,14 @@ def journal_entry(  # noqa: C901, CCR001
         # add mandatory StarSystem, StarPos and SystemAddress properties to Scan events
         if 'StarSystem' not in entry:
             if not system:
-                logger.warning("system is None, can't add StarSystem")
+                logger.warning("system is falsey, can't add StarSystem")
                 return "system is None, can't add StarSystem"
```

Comment:
`is None` here is a problem as noted in the above line, it may or may not actually be None